.evergreen/atlas/setup-atlas-cluster.sh

```diff
@@ -25,6 +25,7 @@ DRIVERS_ATLAS_GROUP_ID
 DRIVERS_ATLAS_LAMBDA_USER
 DRIVERS_ATLAS_LAMBDA_PASSWORD
 LAMBDA_STACK_NAME
+task_id
```

Comment:
although this doesn't follow the existing all-caps convention, I chose to use the same name as the default evergreen expansion because drivers can then rely on `add_expansions_to_env` to ensure the task id is present.

Node, for example, already uses `add_expansions_to_env` so no changes are necessary 😄